diff --git a/shell/ev-application.c b/shell/ev-application.c
index c8ec9c5..974b161 100644
--- a/shell/ev-application.c
+++ b/shell/ev-application.c
@@ -1065,8 +1065,8 @@ ev_application_startup (GApplication *gapplication)
           "win.go-backwards",           "<Shift>Page_Up", NULL,
           "win.go-forward",             "<Shift>Page_Down", NULL,
           "win.go-next-page",           "n", NULL,
-          "win.go-previous-page",       "p", NULL,
-          "win.go-back-history",        "<alt>P", NULL,
+          "win.go-previous-page",       "t", NULL,
+          "win.go-back-history",        "<alt>T", NULL,
           "win.go-forward-history",     "<alt>N", NULL,
           "win.sizing-mode::fit-page",  "f", NULL,
           "win.sizing-mode::fit-width", "w", NULL,
