diff --git a/unix/ibus/property_handler.cc b/unix/ibus/property_handler.cc
index d9da392..7463913 100644
--- a/unix/ibus/property_handler.cc
+++ b/unix/ibus/property_handler.cc
@@ -80,7 +80,7 @@ bool GetDisabled(IBusEngine *engine) {
 // On IBus 1.4.x, IBus expects that an IME should always be turned on and
 // IME on/off keys are handled by IBus itself rather than each IME.
 #if IBUS_CHECK_VERSION(1, 5, 0)
-const bool kActivatedOnLaunch = false;
+const bool kActivatedOnLaunch = true;
 #else
 const bool kActivatedOnLaunch = true;
 #endif  // IBus>=1.5.0
